instance/admin.py

```diff
@@ -67,19 +68,9 @@ class OpenEdXInstanceAdmin(admin.ModelAdmin): # pylint: disable=missing-docstrin
     list_display = ('internal_lms_domain', 'name', 'created', 'modified',
                     'successfully_provisioned')
     search_fields = ('internal_lms_domain',)
-    inlines = [
+    inlines = (
         InlineInstanceReferenceAdmin,
-    ]
-
-    def get_inline_instances(self, request, obj=None):
-        """
-        Hides inlines while creating ``OpenEdXInstance``.
-        """
-        # Doesn't show the inline instance for new objects since we have custom
-        # logic for creating InstanceReference objects
```

Comment:
Is this check no longer needed? It might not be required anymore, but we need to double check that removing this doesn't break anything.

Reply:
I think you were right about this. I misinterpreted "hide" with "collapse".